### PR TITLE
Add Template.registerHelpers

### PR DESCRIPTION
**Pull request by zimme**
As discussed here meteor/meteor#4367

### CODE REVIEW

**Comment by zimme** [comment]
Oh sorry, no tests added. I'll get back to that later this week, don't have the time right now :cry:

**Comment by zimme** [comment]
Tests added, ready to be merged =)

File: template-extension.js [hunk 1]
```diff
@@ -46,6 +46,19 @@ Template.onDestroyed = function (callback) {
   globalHooks.destroyed.push(callback);
 };
 
+Template.registerHelpers = function(helpers) {
+  var func, name;
+  var hasProp = {}.hasOwnProperty;
+
+  for (name in helpers) {
+    if (!hasProp.call(helpers, name))
```
Comment (aldeed) [inline template-extension.js]: Why not just `helpers.hasOwnProperty(name)`?

**Comment by aldeed** [comment]
See one comment but otherwise looks mergeable.

File: template-extension.js (hunk 1 above)
Comment (zimme) [inline template-extension.js]: Sorry about that, just made a quick cs to js conversion, I'll fix it =)

**Pull request merged**